package.json

```diff
@@ -46,58 +46,58 @@
     "classnames": "2.2.5",
     "clean-webpack-plugin": "0.1.17",
     "compass-mixins": "0.12.10",
-    "copy-webpack-plugin": "4.1.1",
+    "copy-webpack-plugin": "4.2.0",
     "css-loader": "0.28.7",
     "dom4": "1.8.5",
     "es6-promise": "4.1.1",
-    "eslint": "4.8.0",
+    "eslint": "4.10.0",
     "eslint-plugin-lodash-fp": "2.1.3",
-    "eslint-plugin-promise": "3.5.0",
+    "eslint-plugin-promise": "3.6.0",
     "eslint-plugin-react": "7.4.0",
     "eslint-plugin-standard": "3.0.1",
     "file-loader": "1.1.5",
-    "i18next": "10.0.1",
+    "i18next": "10.0.7",
     "i18next-resource-store-loader": "0.1.2",
-    "js-cookie": "2.1.4",
+    "js-cookie": "2.2.0",
     "keymirror": "0.1.1",
     "lodash": "4.17.4",
     "markdown": "0.5.0",
     "moment": "2.19.1",
     "node-sass": "4.5.3",
+    "patch-package": "^3.6.0",
     "prop-types": "15.6.0",
-    "react": "15.6.1",
-    "react-addons-css-transition-group": "15.6.2",
-    "react-addons-perf": "15.6.0-rc.1",
+    "react": "16.0.0",
     "react-addons-shallow-compare": "15.6.2",
     "react-datetime": "2.10.3",
     "react-dnd": "2.5.4",
     "react-dnd-html5-backend": "2.5.4",
-    "react-dom": "15.6.1",
-    "react-dropzone": "4.2.0",
-    "react-hot-loader": "3.0.0",
-    "react-i18next": "2.0.1",
-    "react-loader": "2.4.2",
+    "react-dom": "16.0.0",
+    "react-dropzone": "4.2.1",
+    "react-hot-loader": "3.1.1",
+    "react-i18next": "6.1.0",
+    "react-loader": "https://github.com/Ross146/react-loader/tarball/master",
```

Comment:
It is necessary because of react >= 16?

Reply:
It links to a pull request which modifies react-loaders package.json to allow react >= 16